exporter/metric/metric.go

```diff
@@ -74,6 +74,41 @@ type metricExporter struct {
 	startTime time.Time
 }
 
+// Below are maps with monitored resources fields as keys
+// and OpenTelemetry resources fields as values
+var k8sContainerMap = map[string]string{
```

Comment:
- I don't see k8sClusterMap here. Is there any reason you exclude that one? 
- Looks like you impose a strict matching of these resource types (no field should be missing). Is there any reason to throw away partial labels? For example, if we only have "location", "cluster_name" and "pod_name" we can't match this with any predefined resource labels and would return empty. Is it expected?

Reply:
1) k8sClusterMap would be included in the next version.
2) I think we should follow the instruction strictly and discard partial labels because
i. according to the definition of labels in MonitoredResource(https://github.com/googleapis/go-genproto/blob/master/googleapis/api/monitoredres/monitored_resource.pb.go#L217), the Labels map is required to contain the values for all the resources label descriptors. In other words, all keys must exist. 
ii. Otherwise, if we allow missing labels, it would greatly increase the difficulty of subdividing the resources types since we need to consider all possible cases of missing label keys.